### PR TITLE
Engdocs 1268

### PR DESCRIPTION
**Pull request by aevesdocker**
PR cleans up content in automated builds section

### CODE REVIEW

File: docker-hub/builds/manage-builds.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+> **Note**
+>
+> Automated builds requires a
+> [Docker Pro, Team, or Business subscription](../../subscription/index.md).
+
+## Cancel or retry a build
+
+While a build is in queue or running, a **Cancel** icon appears next to its build
+report link on the **General** tab and on the **Builds** tab. You can also select
+**Cancel** on the build report page, or from the **Timeline** tab's logs
```
Comment (craig-osterhout) [inline docker-hub/builds/manage-builds.md:15]: The page name is formatted as a UI control in the troubleshoot topic. I tried looking for this page in the UI, but I'm unsure how to get to it to verify.

```suggestion
**Cancel** on the **Build report** page, or from the **Timeline** tab's logs
```

File: docker-hub/builds/index.md [hunk 2]
```diff
@@ -407,7 +272,7 @@ set of repositories required for a specific build.
 
 If you are building repositories with linked private submodules (private
 dependencies), you also need to add an override `SSH_PRIVATE` environment
-variable to automated builds associated with the account.
+variable to automated builds associated with the account. For more information, see [Troubleshoot](troubleshoot.md#build-repositories-with-linked-private-submodules)
```
Comment (craig-osterhout) [inline docker-hub/builds/index.md:275]:
```suggestion
variable to automated builds associated with the account. For more information, see [Troubleshoot](troubleshoot.md#build-repositories-with-linked-private-submodules).
```

File: docker-hub/builds/link-source.md [hunk 3]
```diff
@@ -49,43 +45,7 @@ organizations.
 
 5. Click **Authorize docker** to save the link.
```
Comment (craig-osterhout) [inline docker-hub/builds/link-source.md:46]:
```suggestion
5. Select **Authorize docker** to save the link.
```

File: docker-hub/builds/troubleshoot.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+title: Troubleshoot your builds
+description: How to troubleshoot automated builds
+keywords: docker hub, troubleshoot, automated builds, autobuilds
+---
+
+> **Note**
+>
+> Automated builds requires a
```
Comment (craig-osterhout) [inline docker-hub/builds/troubleshoot.md:9]: Same as previous comment.
The plural here is awkward to me. If it were a feature name, like "The Automated Builds feature requires ...", then it reads okay. As is, maybe change to "Automated builds require ..."

**Comment by aevesdocker** [comment]
Thank you for the thorough review @craig-osterhout. Not my best work - I'd clearly lost the will to live as the PR went on!

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**